site/views/examples/example/form.njk

```diff
@@ -5,7 +5,7 @@ title: Example with form
 <uip-root>
   <uip-snippets class="uip-toolbar"></uip-snippets>
 
-  <script type="text/html" uip-snippet uip-isolated label="Example with form">
+  <script type="text/html" uip-snippet label="Example with form">
```

Comment:
It was the only example of an isolated playground if I remember correctly...
But non critical we should rework our demo site and examples anyway